Content.Server/Hands/Systems/HandsSystem.cs

```diff
@@ -204,6 +205,16 @@ public bool ThrowHeldItem(EntityUid player, EntityCoordinates coordinates, float
 
             var throwStrength = hands.ThrowForceMultiplier;
 
+            if (_timing.CurTime < hands.LastThrow + hands.TimeToPerfectThrow)
+            {
+                var timeSinceLast = _timing.CurTime - hands.LastThrow;
+                var inaccuracyModifier = 1 - (timeSinceLast / hands.TimeToPerfectThrow);
+                inaccuracyModifier *= inaccuracyModifier;
+                var maxInaccuracy = hands.InaccuracyAfterThrow * inaccuracyModifier;
+                var totalInaccuracy = _random.NextAngle(-maxInaccuracy, maxInaccuracy);
+                direction = totalInaccuracy.RotateVec(direction);
+            }
```

Comment:
You're doubling the throwing angle so 180 degrees actually means you can throw things behind you as it's 180 degrees either direction.